Tasks/Gradle/CodeAnalysis/SonarQube/common.ts

```diff
@@ -6,13 +6,15 @@ import fs = require('fs');
 
 import {ToolRunner} from 'vsts-task-lib/toolrunner';
 
+import tl = require('vsts-task-lib/task');
```

Comment:
I think we should take the oportunity and get rid of common.ts entirely and replace it with a SonarQube orchestrator. It should probably have only 2 public methods - one for configuring the tool runner and one for doing all of the post processing work (report, build breaker)